src/items/repositories/item.repository.ts

```diff
@@ -84,6 +84,18 @@ export class ItemRepository extends Repository<Item> {
       .orderBy('item.createdAt', 'DESC')
       .getOne();
   }
+  async listAnItem(itemId: string, price: number): Promise<Item> {
+    await this.update(
+      {
+        itemId,
+      },
+      { price: String(price), status: 1 }
+    );
+
+    return await this.findOne({
```

Comment:
Minor: You could return the promise without awaiting, since you are not using anything from the promise response.

Reply:
Sure